src/server.js

```diff
@@ -2,7 +2,7 @@ import express from 'express';
 import {} from 'dotenv/config';
 import mongoose from 'mongoose';
 import bodyParser from 'body-parser';
```

Comment:
You don't need to install &/ use body-parser. If you want to accept JSON then you can call express.json() directly. Body parser is now bundled with express by default.